third_party/move/move-compiler-v2/tests/bytecode-verify-failure/equality.exp

```diff
@@ -15,33 +15,8 @@ B0:
 ============ bytecode verification failed ========
 
 Diagnostics:
-bug: BYTECODE VERIFICATION FAILED
+bug: bytecode verification failed with unexpected status code `EQUALITY_OP_TYPE_MISMATCH_ERROR`.
```

Comment:
Could we say something more actionable here? It may not be very clear to the user that this is infact a compiler bug, and not a bug in their code (from just looking at how this error is presented).

Perhaps add: "This is a compiler bug, please help us improve the compiler by reporting this issue [link]", or similar.

Reply:
Done